### ISSUE TITLE
/knora.json doesn't work with a CSV file

### ISSUE THREAD

**Issue by benjamingeer** [dasch-swiss/sipi#309]
Steps to reproduce:

1. Put a CSV file `test.csv` in `/sipi/images/tmp`.
2. Make a request to `http://localhost:1024/tmp/test.csv/knora.json`.

Sipi responds with an internal server error:

```
Internal Server Error: Sipi image error at [/sipi/src/SipiImage.cpp: 381]: Could not read file /sipi/images/tmp/test.csv
```

The Sipi log says:

```
sipi            | Sipi: ERROR IN TIFF! Module: /sipi/images/tmp/test.csv
sipi            | Sipi: Not a TIFF or MDI file, bad magic number 28755 (0x7053)
sipi            | Sipi: GET /tmp/test.csv/knora.json failed (Internal Server Error): Sipi image error at [/sipi/src/SipiImage.cpp: 381]: Could not read file /sipi/images/tmp/test.csv
```

**Comment by benjamingeer** [comment]
[test.csv.zip](https://github.com/dasch-swiss/sipi/files/3755496/test.csv.zip)

**Comment by benjamingeer** [comment]
Ping (this is still blocking https://github.com/dasch-swiss/knora-api/issues/1532)

**Comment by benjamingeer** [comment]
Still not working.

**Comment by lrosenth** [comment]
Yeah – I just fixed it. .../knora.json just returns the mimetype:

```http://localhost:1024/images/hello.gresource.xml/knora.json```

results in:

```json
{
   "internalMimeType": "text/xml"
}
```

**Comment by benjamingeer** [comment]
That's perfect, thanks, can we have a new Sipi release with this fix?

**Comment by benjamingeer** [comment]
It would also be great to have a test for it in https://github.com/dasch-swiss/sipi/blob/develop/test/e2e/test_02_server.py.

**Comment by lrosenth** [comment]
done!

**Comment by benjamingeer** [comment]
I'm still getting the same error with the current Sipi.

**Comment by benjamingeer** [comment]
Now Sipi's log says:

```
Sipi: GET /tmp/test.csv/knora.json failed (Internal Server Error): Sipi image error at [/sipi/src/SipiImage.cpp: 383]: Could not read file /sipi/images/tmp/test.csv
```

But I can see that the file is there in the filesystem.

**Comment by benjamingeer** [comment]
I've just tried this again with the current Sipi, and got the same result.

**Comment by benjamingeer** [comment]
Fixed in `3.0.0-rc.5`.
